Admin/ArticleAdmin.php

```diff
@@ -0,0 +1,301 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\ArticleBundle\Admin;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping\ClassMetadataInfo;
+use Sonata\AdminBundle\Admin\AbstractAdmin;
+use Sonata\AdminBundle\Admin\AdminInterface;
+use Sonata\AdminBundle\Datagrid\ListMapper;
+use Sonata\AdminBundle\Form\FormMapper;
+use Sonata\AdminBundle\Show\ShowMapper;
+use Sonata\ArticleBundle\Model\Article;
+use Sonata\ArticleBundle\Model\Fragment;
+use Sonata\CoreBundle\Validator\ErrorElement;
+use Sonata\MediaBundle\Admin\ORM\MediaAdmin;
+
+/**
+ * @author Florent Denis <florent.denis@ekino.com>
+ */
+class ArticleAdmin extends AbstractAdmin
```

Comment:
> If we set it final, I can already make the issue because I'm gonna need to extends it.

What do you need to extends exactly? Maybe we can make it configurable or make only some method final.

> Maybe we can rename it `BaseArticle` or make it Abstract if you prefer to force the developer extends it makes more sens to me.

=> `AbstractArcticleAdmin`.

Can make sense, I don't know. In this case like I said before, we can make some of the methods final. :wink: